flare_dart/lib/dependency_sorter.dart

```diff
@@ -55,9 +50,10 @@ class DependencySorter {
 
 /// Sorts dependencies for Actors even when cycles are present
 ///
-/// Any nodes that form part of a cycle can be found in `cycleNodes` after `sort`.
-/// NOTE: Nodes isolated by cycles will not be found in `_order` or `cycleNodes`
-///   e.g. `A -> B <-> C -> D` isolates D when running a sort based on A
+/// Any nodes that form part of a cycle can be found in `cycleNodes` after
+/// `sort`. NOTE: Nodes isolated by cycles will not be found in `_order` or
+/// `cycleNodes` e.g. `A -> B <-> C -> D` isolates D when running a sort based
+/// on A
```

Comment:
Sorry for this churn, my analyzer was complaining about the 80 character length limit. I ran my comment formatter on it in VSCode.